nexus_constructor/model/attribute.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+import attr
+
+
+@attr.s
+class FieldAttribute:
+    """
+    Class for containing attributes in the nexus structure.
+    Uses the long format (dict with "name" and "values" as keys)
+    rather than the short form (name:value) as this works for everything and is more flexible, with
+    the trade-off being a longer message (which is not a priority)
```

Comment:
Need to mention the file writer JSON format to make this comment less confusing.